index.d.ts

```diff
@@ -134,75 +168,234 @@ export type WellKnownName = 'Circle' | 'Square' | 'Triangle' | 'Star' | 'Cross'
                             | 'shape://times' | 'shape://oarrow' | 'shape://carrow' ;
 
 /**
- * MarkSymbolizer describes the style representation of POINT data, if styled as with
- * a regular geometry.
+ * MarkSymbolizer describes the style representation of POINT data, if styled as
+ * with a regular geometry.
  */
 export interface MarkSymbolizer extends BasePointSymbolizer {
   kind: 'Mark';
+  /**
+   * The WellKnowName of the MarkSymbolizer.
+   */
   wellKnownName: WellKnownName;
+  /**
+   * The radius of the Symbolizer. Values describing the full size of the symbolizer
+   * have to be divided by two.
+   */
   radius?: number;
+  /**
+   * The rotation of the Symbolizer in degree. Value should be between 0 and 360.
+   */
   rotate?: number;
+  /**
+   * The color of the stroke represented as a hex-color string.
+   */
   strokeColor?: string;
+  /**
+   * The opacity of the stroke. A value between 0 and 1.
+   * 0 is none opaque and 1 is full opaque.
+   */
   strokeOpacity?: number;
+  /**
+   * The width of the stroke in pixel.
+   */
   strokeWidth?: number;
+  /**
+   * Amount to blur the symbolizer. 1 blurs the symbolizer such that only the
+   * centerpoint has full opacity. Mostly relevant for circles.
+   */
   blur?: number;
+  /**
+   * Property relevant for mapbox-styles.
+   * Compare https://docs.mapbox.com/mapbox-gl-js/style-spec/#paint-circle-circle-pitch-alignment
+   */
   pitchAlignment?: 'map' | 'viewport';
+  /**
+   * Property relevant for mapbox-styles.
+   * Compare https://docs.mapbox.com/mapbox-gl-js/style-spec/#paint-circle-circle-pitch-scale
+   */
   pitchScale?: 'map' | 'viewport';
 }
 
-
 /**
- * The TextSymbolizer describes the style representation of point data, if styled with
+ * The TextSymbolizer describes the style representation of point data, if styled
+ * with
  * a text.
  */
 export interface TextSymbolizer extends BasePointSymbolizer {
   kind: 'Text';
+  /**
+   * If true, the text will be visible even if it collides with other previously
+   * drawn symbols.
+   */
   allowOverlap?: boolean;
   anchor?: 'center' | 'left' | 'right' | 'top' | 'bottom' | 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
```

Comment:
Missing comment. Could be: 
```suggestion
  /**
   * The anchor position of the label referred to the center of the geometry. 
   */
  anchor?: 'center' | 'left' | 'right' | 'top' | 'bottom' | 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
```